process-migration-service/examples/persistence/h2.yml

```diff
@@ -0,0 +1,15 @@
+thorntail:
+  datasources:
+    data-sources:
+      pimDS:
+        driver-name: h2
+        connection-url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+        user-name: sa
+        password: sa
+  ejb3:
+    timer-service:
+      default-data-store: timers-store
+      database-data-stores:
+        timers-store:
+          datasource-jndi-name: java:jboss/datasources/pimDS
+          partition: timer
```

Comment:
The ejb3 config points to a datasource. If we do not define persistence then the datasource will not be defined. Maybe an alternative is to default to h2 datasource and have the other persistence configurations override the `pimDS` datasource. WDYT?

Reply:
I meant that it would be enough to have one EJB timer example and just leave there `java:jboss/datasources/pimDS` and just add a comment note that of course you have to have a datasource configured. I think it is obvious that if you don't have a datasource defined and you are referencing it in the database-data-stores, you will have some troubles :)